man/homectl.xml

```diff
@@ -376,6 +376,8 @@
         <para>Note that FIDO2 devices suitable for this option must implement the
         <literal>hmac-secret</literal> extension. Most current devices (such as the YubiKey 5 series) do. If
         the extension is not implemented the device cannot be used for unlocking home directories.</para>
+          
+        <para>The FIDO2 device may be subsequently removed by setting the device path to an empty string</para>
```

Comment:
this is not helpful. Provide the full command, otherwise this is very confusing. Also, please end the sentence in a full stop.

Reply:
Thank you, it wasn't clear to me if you wanted the complete option or complete command.  Final commit is the complete command.